include/Options/Business/Sync.php

```diff
@@ -40,14 +40,11 @@ public static function key(): string {
 	 * @phpstan-return array{type: 'array', items: array{type: SchemaType, enum: non-empty-list<non-falsy-string>}}
 	 */
 	protected function get_data_structure(): array {
-		add_filter( 'lang_dir_for_domain', '__return_false' ); // Prevents loading the translations too early.
-		$enum = array_keys( \PLL_Settings_Sync::list_metas_to_sync() );
-		remove_filter( 'lang_dir_for_domain', '__return_false' );
 		return array(
 			'type'  => 'array',
 			'items' => array(
 				'type' => $this->get_type(),
-				'enum' => $enum,
```

Comment:
As always, I'd like to avoid to duplicate code. Using `unload_textdomain()` added to the previous code should do the trick. Am I wrong?

Reply:
I will check.

Reply:
> I will check.

It's ok.